commander/src/utils/constants.ts

```diff
@@ -49,9 +47,16 @@ export const API_PROTOCOLS: ReadonlyArray<string> = ['http:', 'https:'];
 
 export const COMMUNITY_IDENTIFIER = 'Lisk';
 
+// FIXME: Update this based on new network ID input
 export const NETHASHES: { readonly [key: string]: string } = {
-	main: MAINNET_NETHASH,
-	test: TESTNET_NETHASH,
+	main: Buffer.from(
+		'ed14889723f24ecc54871d058d98ce91ff2f973192075c0155ba2b7b70ad2511',
+		'hex',
+	).toString('base64'),
+	test: Buffer.from(
+		'da3ed6a45429278bac2666961289ca17ad86595d33b31037615d4b8e8f158bba',
+		'hex',
+	).toString('base64'),
```

Comment:
This seems interesting. Why no we just use `base64` strings and try to remember those onward. May be to recall better, just add hex string as a comment on top of those.

Reply:
This nethash, network identifier will need to be updated, so that's why I kept it as it is for now.
Nethash will not be used in the network identifier